Assignment2.java

```diff
@@ -0,0 +1,125 @@
+                
+            }else if(input == 2){
+                
+                //IN CLASS PROJECT
+                Double yourPoints;
+                Double maxPoints;
+                Double score;
+                char grade;
+                
+                 System.out.println("GRADE CALCULATOR");
+                 System.out.print("\tEnter YOUR SCORE: ");
+                 yourPoints = keyboard.nextDouble(); 
+                 System.out.print("\tEnter MAX POINTS: ");
+                 maxPoints = keyboard.nextDouble(); 
+                 score = (yourPoints/maxPoints) * 100;
+                
+                 if(score >= 90) {
+                        grade = 'A';
+                 } else if (score >= 80) {
+                     grade = 'B';
+                 } else if (score >= 70) {
+                     grade = 'C';
+                 } else if (score >= 60) {
+                     grade = 'D';
+                 } else {
+                     grade = 'F';
+                 };
+                 
+                 System.out.println("\tYour grade: " + score + " " + grade);
+                 
+            }else if(input == 3){
+                
+              //LOOPING LYRICS
+                Integer numLoops;
+                String[] lyrics = {
+                  "Adventure Time,", "Come on, grab your friends", "We'll go to" , 
+                    "very distant lands", "With Jake the Dog", "And Finn the Human",
+                    "The fun will never end,", "It's Adventure Time!"
+                };    
+                
+                //TITLE AND DESCRIPTION
+                System.out.print("SONG LYRICS LOOP\n" + "\tEnter a number from"
+                        + " 1-8. This will be the number of times the loop"
+                        + " executes: ");
+                numLoops = keyboard.nextInt();
+                
+                //validate entry
+                while (numLoops != 9) {
+                    if(numLoops <= 8 && numLoops >= 0) {
+                        for(i=0; i < numLoops; i++) {
+                            System.out.println("\t\t" + lyrics[i]);
+                        }
+                    }else{
+                        System.out.println("Oops! Try again");
+                    } 
+                    System.out.print("\nEnter a number 1-8 to loop again or 9 to"
+                            + " return to main menu: ");
+                    numLoops = keyboard.nextInt();
+                }
+            } else if (input == 4) {
+                System.out.println("Programmer is furiously typing away...");
+            } else if (input == 5) {
+                System.out.println("Programmer is furiously typing away...");
+            }else if (input == 6){  
+                bool = false;
+            }else{
+                System.out.println("Oops! Try again.");   
+            } 
+        }
+        System.out.println("Goodbye!");
+    };
+};
```

Comment:
Nice! I like your menu items array and loop. I think it could use a few more comments, though.